guide/guide.md

```diff
@@ -101,6 +101,45 @@ Do not put any spaces around the `.`.
 x.foo().bar().baz(x, y, z);
 ```
 
+### Match
+
+For a single-expression matcher to a single-expression matchee, which fits on
```

Comment:
I'm unsure how to write this better. "single-expression matcher" means `Foo(x) =>` as opposed to `Foo(x) | Bar(x) =>`. "Single expression matchee" means `=> x,`